app/views/groups/_group.html.erb

```diff
@@ -0,0 +1,13 @@
+<div id="<%= dom_id group %>">
+<%= link_to group_spendings_path(group.id), class:"text-decoration-none text-dark shadow" do %>
+ <div class="d-flex justify-content-between align-items-center bg-light m-3 p-2 single-group-card">
+
+      <%= image_tag group.icon, :size => '100x100', id: "group_icon" %>
```

Comment:
- [ ] You might need to check the existance of this icon in your asset pipeline before rendering it here, so that your app doesn't crash when a user enters an invalid or no-existing image, for example in my case, I have set a wrong image link and now see how the app looks like:
![Screenshot from 2022-02-16 23-41-50](https://user-images.githubusercontent.com/56886064/154362716-8fd957f6-1bb1-436d-9ef9-abe622faa705.png)

Reply:
This issue is well resolved! ✔️